src/pages/Releases/ReleasesPage.js

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import marked from "marked";
+import { MuiThemeProvider, withStyles } from "@material-ui/core/styles/index";
+import DOMPurify from "dompurify";
+import classNames from "classnames";
+
+import Paper from "@material-ui/core/Paper";
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableRow from "@material-ui/core/TableRow";
+import TableCell from "@material-ui/core/TableCell";
+import TableFooter from "@material-ui/core/TableFooter";
+import TablePagination from "@material-ui/core/TablePagination";
+import Card from "@material-ui/core/Card";
+import CardContent from "@material-ui/core/CardContent";
+
+import useReleases from "../../hooks/useReleases";
+import Header from "../../components/Header/Header";
+import CONSTANTS from "../../helpers/constants";
+
+const styles = () => ({
+    container: {
+        padding: "6rem"
+    },
+    cardHolder: {
+        border: "none",
+        paddingBottom: "1rem"
+    },
+    isFirstCard: {
+        borderWidth: "2px",
+        borderStyle: "solid",
+        borderColor: "#9ccc65",
+        backgroundColor: "#f1f8e9"
+    }
+});
+
+const ReleasesPage = (props) => {
+    const { classes, gitHubURL } = props;
+    const versions = useReleases(gitHubURL);
+    const [page, setPage] = useState(0);
+    const [rowsPerPage, setRowsPerPage] = React.useState(4);
+
+    function getMarkdownText(body) {
+        const rawMarkup = marked(DOMPurify.sanitize(body), { sanitize: true });
+        return { __html: rawMarkup };
+    }
+
+    const getReleaseDetails = version => {
+        return (
+            <div key={version.id} dangerouslySetInnerHTML={getMarkdownText(version.body)}/> //  eslint-disable-line react/no-danger
+        );
+    };
+
+    const handlePageChange = (event, newPage) => setPage(newPage);
+    const handleChangeRowsPerPage = (event) => {
+        setRowsPerPage(parseInt(event.target.value, 10));
+        setPage(0);
+    };
+
+    if (!versions.length) {
+        return "";
+    }
+
+    return (
+        <MuiThemeProvider theme={CONSTANTS.THEME()}>
+            <Paper>
+                <Header withNav={false}/>
+                <div className={classes.container}>
+                    <Table>
+                        <TableBody>
+                            {versions.slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage).map((v, index) => (
+                                <TableRow key={v.id}>
+                                    <TableCell className={classes.cardHolder}>
+                                        <Card raised={index === 0} className={classNames({ [classes.isFirstCard]: index === 0 })}>
```

Comment:
Instead of matching the first card, we would want to match the first `release`. I think you can match the object in `versions[0]` instead?

Reply:
https://github.com/webpack/webpack/releases

Reply:
+1. So the first item that will have both `prerelease` and `draft` as `false` will be picked as latest release